install-acts-as-list

From: Bryan Larsen <bryan@larsen.st>

# Task re-ordering

We're now going to add the ability to re-order a story's tasks by drag-and-drop. There's support for this built into Hobo, so there's not much to do. First we need the `acts_as_list` plugin:

    $ ./script/plugin install acts_as_list

or

    $ git submodule add git://github.com/rails/acts_as_list.git vendor/plugins/acts_as_list
---

 .gitmodules                 |    3 +++
 vendor/plugins/acts_as_list |    1 +
 2 files changed, 4 insertions(+), 0 deletions(-)
 create mode 100644 .gitmodules
 create mode 160000 vendor/plugins/acts_as_list


diff --git a/.gitmodules b/.gitmodules
new file mode 100644
index 0000000..d22460d
--- /dev/null
+++ b/.gitmodules
@@ -0,0 +1,3 @@
+[submodule "vendor/plugins/acts_as_list"]
+	path = vendor/plugins/acts_as_list
+	url = git://github.com/rails/acts_as_list.git
diff --git a/vendor/plugins/acts_as_list b/vendor/plugins/acts_as_list
new file mode 160000
index 0000000..8771a63
--- /dev/null
+++ b/vendor/plugins/acts_as_list
@@ -0,0 +1 @@
+Subproject commit 8771a632dc26a7782800347993869c964133ea29
